pom.xml

```diff
@@ -107,29 +110,36 @@
             <version>8.10.2</version>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.sun.activation</groupId>
+            <artifactId>jakarta.activation</artifactId>
+            <version>1.2.1</version>
+        </dependency>
```

Comment:
do we actually need this? i don't think we use it explicity

Reply:
we use it explicitly, in getting the data handler class